packages/types/src/chain.ts

```diff
@@ -58,6 +59,7 @@ export type ChainAddresses = {
   W_TOKEN_USD_CHAINLINK_PRICE_FEED: string;
   UNISWAP_V2_ROUTER: string;
   UNISWAP_V2_FACTORY: string;
+  UNISWAP_V3_ROUTER?: string;
```

Comment:
I suggest we use something like this: https://github.com/Midas-Protocol/monorepo/blob/feature/security/packages/types/src/chain.ts

Reply:
Because each chain will also likely need as specific PAIR_INIT_HASH and quoter addresses

Reply:
we don't need `pair_init_hash` for v3 and we can get `quoter` address from deployments.